file_converter/Program.cs

```diff
@@ -2,94 +2,73 @@
 using System.Linq;
 
 namespace file_converter
-{   
+{
     class Program
     {
+        private static string[] SUPPORTED_FORMATS = { "txt", "csv", "json", "xml" };
+        private static string GetFileExtension(string filePath)
+        {
+            string[] s = filePath.Split("."); 
+            return (s[s.Length-1]);
+        }
+        private static IFileWrapper BuildFileWrapper(string filePath)
+        {
+            string ext = GetFileExtension(filePath);
+            switch (ext)
+            {
+                case "txt":
+                    return new fileTxt(filePath);                   
+                case "csv":
+                    return new fileCsv(filePath);                   
+                case "json":
+                    return new fileJson(filePath);                   
+                case "xml":
+                    return new fileXml(filePath);
+            }
+            return new fileTxt(filePath);
+        }
         static void Main(string[] args)
```

Comment:
`Main` method should be above all other methods. This is not a strict rule, but a recommendation.